api/app/tag.py

```diff
@@ -246,11 +249,42 @@ def make_image(payload: bytes, frame_type: Literal["simple", "secure"]) -> bytes
         image_factory=StyledPilImage,
     )
 
-    # TODO: Wrap in frame using cairosvg + pillow
+    # Get the dimensions of the image in pixels
+    img_width, img_height = img.size
 
-    # Get image contents as bytes
+    # Convert the SVG frame to a PNG image
+    api_root = Path(__file__).parent.parent.absolute()
+    frame_path = str(api_root) + "/tags_frame_signed.svg" if frame_type == "secure" else str(
+        api_root) + "/tags_frame_simple.svg"
+    with open(frame_path, "rb") as svg_file:
+        svg_data = svg_file.read()
+        png_data = cairosvg.svg2png(bytestring=svg_data)
```

Comment:
You will likely need to add arguments here to get the correct resolution from this stage 
![image](https://github.com/ioxiocom/ioxio-tags/assets/1296558/64452db7-c96c-4327-85a2-e30b9f97e16c)

Reply:
And the background color